src/main/java/com/artipie/asto/s3/S3Storage.java

```diff
@@ -67,19 +57,14 @@
 /**
  * Storage that holds data in S3 storage.
  *
- * @since 0.15
  * @todo #87:60min Do not await abort to complete if save() failed.
  *  In case uploading content fails inside {@link S3Storage#save(Key, Content)} method
  *  we are doing abort() for multipart upload.
  *  Also whole operation does not complete until abort() is complete.
  *  It would be better to finish save() operation right away and do abort() in background,
  *  but it makes testing the method difficult.
- * @todo #193:30min Too many methods in `S3Storage` class.
- *  `S3Storage` has too many methods.
- *  It could be refactored to move some of it's logic to other classes.
- *  In particular there three methods related to saving bytes
- *  (multipart or not, selecting proper method). This could be moved out of this class.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @since 0.15
  */
 @SuppressWarnings("PMD.TooManyMethods")
 public final class S3Storage implements Storage {
```

Comment:
@andreoss is it possible to remove now this annotation?

Reply:
@genryxy Done